tests/locators/hw22.spec.js

```diff
@@ -0,0 +1,543 @@
+            // Открыть форму регистрации нового пользователя
+            await page.goto("/")
+            const signUpBtn = page.locator(".hero-descriptor_btn")
+            await signUpBtn.click()
+            // Как эти три строки можно вынести в beforeEach? Почему ругается на /
+
+            const form = page.locator('app-signup-modal form')
+            const nameInput = form.locator('#signupName')
+
+            await nameInput.fill("Aa")
+            await nameInput.blur()
+
+            const registerBtn = page.locator(".modal-footer > .btn")
+            await expect(registerBtn).toBeDisabled()
+        })
+
+        test("Name field > Name is correct: long max = 20 symbols", async ({page}) => {
+            // Открыть форму регистрации нового пользователя
+            await page.goto("/")
+            const signUpBtn = page.locator(".hero-descriptor_btn")
+            await signUpBtn.click()
+            // Как эти три строки можно вынести в beforeEach? Почему ругается на /
+
+            const form = page.locator('app-signup-modal form')
+            const nameInput = form.locator('#signupName')
+
+            await nameInput.fill("Aaaaaaaaaaaaaaaaaaaa")
+            await nameInput.blur()
+
+            const registerBtn = page.locator(".modal-footer > .btn")
+            await expect(registerBtn).toBeDisabled()
+        })
+
+        test("Name field > Empty field", async ({page}) => {
+            // Открыть форму регистрации нового пользователя
+            await page.goto("/")
+            const signUpBtn = page.locator(".hero-descriptor_btn")
+            await signUpBtn.click()
+            // Как эти три строки можно вынести в beforeEach? Почему ругается на /
+
+            const form = page.locator('app-signup-modal form')
+            const nameInput = form.locator('#signupName')
+
+            await nameInput.focus()
+            await nameInput.blur()
+
+            const inputNameTip = page.locator('.invalid-feedback > p')
+            await expect(inputNameTip).toContainText('Name required')
+            await expect(nameInput).toHaveCSS('border-color', 'rgb(220, 53, 69)')
+
+            const registerBtn = page.locator(".modal-footer > .btn")
+            await expect(registerBtn).toBeDisabled()
+        })
+
+        test("Name field > Name is invalid: Numbers", async ({page}) => {
+            // Открыть форму регистрации нового пользователя
+            await page.goto("/")
+            const signUpBtn = page.locator(".hero-descriptor_btn")
+            await signUpBtn.click()
+            // Как эти три строки можно вынести в beforeEach? Почему ругается на /
+
+            const form = page.locator('app-signup-modal form')
+            const nameInput = form.locator('#signupName')
+
+            await nameInput.fill("111")
+            await nameInput.blur()
+
+            const inputNameTip1 = page.locator('.invalid-feedback > :nth-child(1)')
```

Comment:
you don't need any numbers in your variable name

Reply:
in all tests

Reply:
if there is `inputNameTip1` I'd expect to see `inputNameTip2` at least

Reply:
It was by mistake